homeassistant/components/remote/demo.py

```diff
@@ -0,0 +1,63 @@
+"""
+Demo platform that has two fake remotes.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/demo/
+"""
+from homeassistant.components.remote import RemoteDevice
+from homeassistant.const import DEVICE_DEFAULT_NAME
+
+
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+    """Setup the demo remotes."""
+    add_devices_callback([
+        DemoRemote('Remote One', False, None, False),
+        DemoRemote('Remote Two', True, 'mdi:remote', False),
+    ])
+
+
+class DemoRemote(RemoteDevice):
+    """Representation of a demo switch."""
+
+    def __init__(self, name, state, icon, assumed):
```

Comment:
not sure about the current activity, if you think that's worth implementing at the component level rather than Harmony level i can add it in.  I didnt include it in the demo platform since not all platforms may support the current activity